packages/core/src/Link/Link.stories.tsx

```diff
@@ -27,6 +27,12 @@ export function ForwardRefs() {
           I am a link!
         </Link>
         <br />
+        <Link variation='subtle'>subtle</Link>
+        <br />
+        <Link variation='plain'>plain</Link>
+        <br />
+        <Link variation='white'>white</Link>
+        <br />
```

Comment:
Would you mind moving these to a new story that shows all of the variations?

Reply:
@craigpalermo I just added all the variations of the link, you can view it [here](https://5a85e56213417800200978ab-lxiuwcvals.chromatic.com/?path=/story/link--all-link-examples). Please have a look at this PR when you get a chance.